client/src/components/BookList.js

```diff
@@ -9,7 +9,6 @@ import PropTypes from 'prop-types';
 
 import CategoryFilter from './CategoryFilter';
 import Book from './Book';
-import EditModal from './EditModal';
 
 class BookList extends Component {
```

Comment:
At the end of the component, you have added the `ConnectedComponent`. This can be improved in order to separate the ConnectedComponent from the Component itself. 

When you need to add a Connected, should be with this structure:
- MyComponent
  - index.js: In this file, only import the connected component and export it as default. This way you can when you import "MyComponent" the connected component will be imported (when you import a folder, it executes the index.js file by default)
  - HOCMyComponent.js: Here will be the code from line 69 to the end (with the imports that it needs)
  - MyComponent.js: Here you will have the rest of the code.